addons/website_sale_slides/static/src/js/slides_course_quiz.js

```diff
@@ -19,7 +23,48 @@ sAnimations.registry.websiteSlidesQuizNoFullscreen.include({
 
 Quiz.include({
     xmlDependencies: (Quiz.prototype.xmlDependencies || []).concat(
-        ["/website_sale_slides/static/src/xml/website_sale_slides_quiz.xml"]
-    )
+        ["/website_sale_slides/static/src/xml/channel_management.xml"]
+    ),
+});
+
+CourseJoinWidget.include({
```

Comment:
Extract the "CourseJoinWidget" part into a "slides_course_join.js" file.
So that it reflects the split in the base website_slides module.